pl_bolts/models/detection/components/torchvision_backbones.py

```diff
@@ -0,0 +1,102 @@
+from typing import Optional, Tuple
+
+import torch.nn as nn
+
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from pl_bolts.models.detection.components._supported_models import TORCHVISION_MODEL_ZOO
+else:
+    warn_missing_pkg("torchvision")  # pragma: no-cover
```

Comment:
I believe `from pl_bolts.models.detection.components._supported_models import TORCHVISION_MODEL_ZOO` will warn so this `if/else` is unnecessary here.
```suggestion
from pl_bolts.models.detection.components._supported_models import TORCHVISION_MODEL_ZOO
from pl_bolts.utils import _TORCHVISION_AVAILABLE
from pl_bolts.utils.warnings import warn_missing_pkg
```